doc/usage.rst

```diff
@@ -29,6 +29,37 @@ the OpenML Python connector, followed up by a simple example.
 
 * `Introduction <examples/introduction_tutorial.html>`_
 
+~~~~~~~~~~~~~
+Configuration
+~~~~~~~~~~~~~
+
+The configuration file resides in a directory ``.openml`` in the home
+directory of the user and is called config. It consists of ``key = value`` pairs
+which are separated by newlines. The following keys are defined:
+
+* apikey:
+    * required to access the server
+
+* server:
+    * default: ``http://www.openml.org``
+
+* cachedir:
+    * if not given, will default to ``file.path(tempdir(), "cache")``.
```

Comment:
Check what this normally resolves to by default and list that instead. I think it should be `~/.openml/cache`.